src/pages/AuthenticatedLanding/ReviewInstitutions.tsx

```diff
@@ -59,8 +59,8 @@ export function ReviewInstitutions({
       </Heading>
       <Paragraph>
         You are required to provide certain identifying information about your
-        associated financial institutions as part of your submission. Click on
-        your financial institution to view or update your financial institution
+        associated financial institutions as part of your filing. Click on your
```

Comment:
@billhimmelsbach 
This looks good. I am flagging it here because of the word "click." I will add it to a list of things to revisit and change.

Reply:
Yeah, let's take a look at this one in a later task. 👍